_data/accessibility-tests/select.yml

```diff
@@ -0,0 +1,113 @@
+title: Select
+component_status: pass
+test_items:
+# General tests
+  - summary: Select component works on mobile devices.
+    summary_additional: |
+      On mobile devices and tablets,
+      the select component should work in both portrait and landscape orientation with no loss of functionality.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.4
+  - summary: Select text and background meet color contrast requirements.
+    summary_additional: |
+      When you use ANDI or another color contrast analyzer on select,
+      contrast between the text and the background is at least 4.5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.3
+# Zoom/screen magnification tests
+  - summary: Select component remains functional when screen is magnified.
+    summary_additional: |
+      When you view the select and zoom to 400%,
+      the select list remains both visible and functional.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: All features of the select element are keyboard operable.
+    summary_additional: |
+      When using a keyboard,
+      you are able to tab into the select element, arrow down to select an option, and hit enter to select.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.1
+  - summary: The focus indicator appears around every element in the select component.
+    summary_additional: |
+      When navigating to select,
+      you can see the focus indicator appear around the select component and on options within the select dropdown.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.7
+  - summary: Navigating to select doesn't trigger a change of context.
+    summary_additional: |
+      When you type into the select,
```

Comment:
Wondering if there's a better way to use this since generally users don't "type into" a select component. That said, users can navigate through the component using character keys, which causes the focus to jump to the first option with that key.

Maybe something like:

> _While interacting with the select component_ the context...."

Reply:
Updated in [65652cb](https://github.com/uswds/uswds-site/pull/2702/commits/65652cb7f057e2a2dc7cc7a6014d6cd16eec4fa0)

> **Navigating to select doesn't trigger a change of context.** When you interact with the select component, the context of the component doesn't change. For example, landing on select doesn't cause a window to open or focus to jump to another field.